notebooks/Add GPS Metadata Fields_FS.ipynb

```diff
@@ -19,16 +19,16 @@
     "### Steps\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #103.    num_manual_locations_field = [field for field in feature_layer_fields if str(field['name']).lower() == 'esrignss_manual_locations']
</pre><p>Is the string casting necessary?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Updated to implicitly cast as suggested.